### PR TITLE
First run of CakePHP code sniffer fixer.

### PR DESCRIPTION
**Pull request by dereuromark**
We must do this incrementally.
There are too many code smells that need manual fixing.

This was the first run of our higher quality cs sniff:

    composer cs-fix

### CODE REVIEW

File: tests/Phinx/Migration/ManagerTest.php [hunk 1]
```diff
@@ -4083,29 +4087,29 @@ public function rollbackToVersionByExecutionTimeDataProvider()
...
                     ],
-                    '20121225000000', 
+                    '20121225000000',
                     'Target version (20121225000000) not found',
                 ],
 
             // Breakpoint set on last migration
 
-            'Rollback to first created version with was also the first to be executed - breakpoint set on last (executed and created) migration' => 
+            'Rollback to first created version with was also the first to be executed - breakpoint set on last (executed and created) migration' =>
```
[the remark this answers is not in the document]
Comment (dereuromark) [reply tests/Phinx/Migration/ManagerTest.php]: Can we silence this maybe?...

File: src/Phinx/Db/Adapter/SQLiteAdapter.php [hunk 2]
```diff
@@ -1030,13 +1035,14 @@ protected function getCommentDefinition(Column $column)
...
         }
+
         return '';
     }
 
     /**
      * Gets the SQLite Index Definition for an Index object.
      *
-     * @param Index $index Index
+     * @param Index $table Index
```
Comment (JayPHP) [inline src/Phinx/Db/Adapter/SQLiteAdapter.php]: Mistake here, due to missing `@param`

**Comment by dereuromark** [comment]
I recommend merging and fixing it piece by piece in small PRs to ease review.

**Comment by JayPHP** [comment]
@dereuromark Merging and fixing what? The changes in this PR do not need to be separated out IMO, there isn't that much to review and it's mainly w/s changes

**Pull request merged**